modules/azure/api_management_api/main.tf

```diff
@@ -47,7 +47,7 @@ resource "azurerm_api_management_api" "api" {
     authorization_server_name = "${lower(replace(var.api_settings.name, " ", "-"))}-auth"
   }
 
-  soap_pass_through = var.soap_pass_through
+  api_type = var.api_type
```

Comment:
indeed this is a breaking change (anyone who uses this module, it will stop working.

Could you keep the variable named: 
soap_pass_through but make it something as follow: 

api = var. api_type == null && var.soap_pass_through? "soap":   var.api_type !=null? var. api_type : "http" 

This way: even if someone